schemas/upgrade.json

```diff
@@ -56,22 +56,14 @@
                 "workflow": "Upgrade",
                 "propertyOrder": 50
         },
-        "force_vsc_standalone_upgrade": {
```

Comment:
This is not a major version? Should we remove it and break schema compatibility? Or should we add depricated for now? @bacastelli can you suggest how do we proceed here?

Reply:
added decrypt and decryption message